lib/jekyll/url.rb

```diff
@@ -93,7 +92,7 @@ def generate_url_from_drop(template)
     # as well as the beginning "/" so we can enforce and ensure it.
 
     def sanitize_url(str)
-      "/" + str.gsub(/\/{2,}/, "/").gsub(%r!\.+\/|\A/+!, "")
+      "/" + str.gsub(/\/{2,}/, "/").gsub(/\.+\/|\A\/+/, "")
```

Comment:
If we kick out `AllowInnerSlashes: true` then it will go on and require you to do %r!! for clarity, sound fine?